cypress/support/commands.js

```diff
@@ -94,3 +94,22 @@ Cypress.Commands.add('testPagination', (selector, test) => {
   }
 })
 
+Cypress.Commands.add('sorting', (selector, order) => {
+  var sortedNames = ""
+  const collator = new Intl.Collator('en-US', {ignorePunctuation: true})
+  
+  cy.get(selector)
+  .then(($cell) => Cypress._.map($cell, (el) => el.innerText))
+  .then((list)=>{
+    if(order == "asc"){
+      sortedNames = list.sort((a, b) => collator.compare(a, b))
+    }else{
+      sortedNames = list.sort((a, b) => collator.compare(b, a))
+    }
```

Comment:
The array needs to be copied with `[...list]` before calling `sort()`, otherwise the original array is modified and you are comparing the array to itself:

```suggestion
  const collator = new Intl.Collator('en-US', {ignorePunctuation: true})
  
  cy.get(selector)
  .then(($cell) => Cypress._.map($cell, (el) => el.innerText))
  .then((list)=>{
    const sortedNames = [...list].sort((a, b) => order === "asc" ? collator.compare(a, b) : collator.compare(b, a));
```

Reply:
@tpapaioa done, what do you think?